Doc/library/math.rst

```diff
@@ -199,8 +199,8 @@ Power and logarithmic functions
 
 .. function:: exp(x)
 
-   Return ``e**x``.
-
+   Return e raised to the power *x*, where e = 2.718281... is the base
```

Comment:
Yes, it's not clear. The problem with marking it up is that we get back to the issue where it could be confused with `math.e`.  I decided not to mark it up, to be consistent with the documentation for `math.e`.